### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Mikey1-nuel**
## Bookstore: initialize project with components

**In this project we**

- Setup a new React application using Create React App
- Created a directory for our reusable components.
- Install React Router V6.
- Added 2 routes in our application and each render a page component:

- Created components inside of the components directory that:
  - Contains individual book state, as well as a button to delete the book
  - Contains a list to renders individual books
  - Contains a form to create new books
  - Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: src/components/nav.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <Link to="/">
+          <li className="navlink" title="home">
+            BOOKS
+          </li>
+        </Link>
+        <Link to="/calculator">
+          <li className="navlink" title="calculator">
+            CATEGORIES
+          </li>
+        </Link>
```
Comment (OyePriscilla) [inline src/components/nav.js]: - [ ] For this page to be rendered correctly, I think the path should be `**categories**` as stated in your routes path `(app.js)`. At the moment, nothing is displayed when I clicked on CATEGORIES link. 👍 

Untitled

File: README.md [hunk 2]
```diff
@@ -1,288 +1,185 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link]()
```
Comment (OyePriscilla) [inline README.md]: - [ ] Kindly fix this live demo link that is redirecting me to a wrong page. If there is none at the moment, you can remove the header. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Mikey1-nuel 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Project initialized using Create React App ✔️ 
Used React Router ✔️ 
Used React components ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
....

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
....

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 ✔️ 

Hi @Mikey1-nuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Used React Router ✔️
- [ ] Correct Gitflow ✔️ 
- [ ] Used React components ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**